ui/widgets/nuxeo-selectivity.js

```diff
@@ -7457,6 +7450,10 @@ typedArrayTags[weakMapTag] = false;
       }
       return scrollParent;
     }
+
+    _computeAriaLabel() {
+      return this.i18n('command.remove');
```

Comment:
Btw, since this method is just the `this.i18n('command.remove')`, why don't you use ti directly in html template?

Reply:
Done